stdlib/public/Observation/Sources/Observation/ObservationRegistrar.swift

```diff
@@ -106,20 +122,35 @@ public struct ObservationRegistrar: Sendable {
     }
   }
   
+  /// A property observation called before setting the value of the subject.
+  ///
+  /// - Parameters:
+  ///     - subject: An instance of an observable type.
+  ///     - keyPath: The key path of an observed property.
   public func willSet<Subject: Observable, Member>(
       _ subject: Subject,
       keyPath: KeyPath<Subject, Member>
   ) {
     context.willSet(subject, keyPath: keyPath)
   }
   
+  /// A property observation called after setting the value of the subject.
+  ///
+  /// - Parameters:
+  ///   - subject: An instance of an observable type.
+  ///   - keyPath: The key path of an observed property.
   public func didSet<Subject: Observable, Member>(
       _ subject: Subject,
       keyPath: KeyPath<Subject, Member>
   ) {
     
   }
   
+  /// Identifies mutations to the transactions registered for observers.
```

Comment:
it might be worth noting that this will just call `willSet`, the mutation, then `didSet`